packages/react-scripts/scripts/eject.js

```diff
@@ -129,7 +129,7 @@ prompt(
   // Add Jest config
   console.log('  Adding ' + cyan('Jest') + ' configuration');
   appPackage.jest = createJestConfig(
-    filePath => path.join('<rootDir>', filePath),
+    filePath => path.posix.join('<rootDir>', filePath),
```

Comment:
> I was looking at "setupFiles": [ "<rootDir>\\config\\polyfills.js" ] from #1417. Doesn't this mean the whole path needs to be fixed?

The problem is that the windows flavor of path.join will switch `/` to `\`, this prevents that from happening. We always pass it `/` so we don't need to fix the whole path.